cmd/insights/commands/collect_test.go

```diff
@@ -0,0 +1,88 @@
+package commands_test
+
+import (
+	"runtime"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/ubuntu/ubuntu-insights/cmd/insights/commands"
+	"github.com/ubuntu/ubuntu-insights/internal/collector"
+	"github.com/ubuntu/ubuntu-insights/internal/testutils"
+)
+
+func TestCollect(t *testing.T) {
+	t.Parallel()
+	tests := map[string]struct {
+		args []string
+
+		consentDir     string
+		removeFiles    []string
+		platformSource bool
+
+		wantErr      bool
+		wantUsageErr bool
+	}{
+		"Collect Basic": {
+			args:           []string{"collect"},
+			platformSource: true,
+		},
```

Comment:
lots of setup and configuration for only one test, did you intend for more tests or should this be made not table based?